lib/SPIRV/libSPIRV/SPIRVType.h

```diff
@@ -183,9 +183,8 @@ class SPIRVTypeInt : public SPIRVType {
     SPIRVEntry::validate();
     assert(BitWidth > 1 &&
            (BitWidth <= 64 ||
```

Comment:
To be precise it should be
```suggestion
    assert((BitWidth == 8 || BitWidth == 16 || BitWidth == 32 || BitWidth == 64 ||
```

Reply:
Done